libpod/oci_linux.go

```diff
@@ -417,6 +427,14 @@ func (r *OCIRuntime) createOCIContainer(ctr *Container, cgroupParent string, res
 		}
 		logrus.Debugf("Received container pid: %d", ss.si.Pid)
 		if ss.si.Pid == -1 {
+			data, err := ioutil.ReadFile(ociLog)
+			if err == nil {
+				var ociErr ociError
+				if err := json.Unmarshal(data, &ociErr); err == nil {
+					return errors.Wrapf(ErrInternal, "container create failed: %s", strings.Trim(ociErr.Msg, "\n"))
```

Comment:
This should not be an ErrInternal, I think we should just return the strings.Trim(ociErr.Msg, "\n")
return errors.Errorf(strings.Trim(ociErr.Msg, "\n"))

Reply:
I'd add a new `ErrOCIRuntime` for errors in the OCI runtime and wrap that